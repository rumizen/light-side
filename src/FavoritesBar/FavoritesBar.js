import React from 'react';
import './FavoritesBar.scss';

const FavoritesBar = ({ faveCount }) => {
  return (
    <button className="favorites-bar">
      <h3>Favorites</h3>
      <p>{faveCount}</p>
    </button>
  );
};

export default FavoritesBar; 