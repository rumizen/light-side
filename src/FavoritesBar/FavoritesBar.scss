.favorites-bar {
  // -webkit-appearance: none;
  background-color: rgb(194, 191, 34);
  display: flex;
  flex-direction: column;
  justify-content: center;
  h3 {
    margin: 0;
  }
  p {
    margin: 0;
  }
}