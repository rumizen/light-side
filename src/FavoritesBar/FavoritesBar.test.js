import React from "react";
import { shallow, mount } from "enzyme";
import FavoritesBar from "./FavoritesBar";

describe("FavoritesBar", () => {
  it("should match the FavoritesBar snapshot", () => {
    const wrapper = shallow(
    <FavoritesBar
      faveCount={5}
    />);

    expect(wrapper).toMatchSnapshot();
  });
});
