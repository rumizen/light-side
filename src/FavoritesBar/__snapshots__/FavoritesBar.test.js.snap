// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesBar should match the FavoritesBar snapshot 1`] = `ShallowWrapper {}`;
